src/Root.tsx

```diff
@@ -0,0 +1,23 @@
+import { Navigate, Route, Routes } from 'react-router-dom';
+
+import { App } from './App';
+import { HomePage } from './components/HomePage';
+import { PageNotFound } from './components/PageNotFound';
+import { PeoplePage } from './components/PeoplePage';
+
+export const Root = () => {
+  return (
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="/home" element={<Navigate to={'/'} />} />
```

Comment:
Your solution doesn't follow requirement #3 from readme 

```suggestion
        <Route path="/home" element={<Navigate to={'/'} replace />} />
```